apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -30,99 +30,72 @@ namespace Google.Cloud.Bigtable.V2
-            if (channelCredentials == null)
-            {
-                var credentials = await GoogleCredential.GetApplicationDefaultAsync().ConfigureAwait(false);
-                if (credentials.IsCreateScopedRequired)
-                {
-                    credentials = credentials.CreateScoped(BigtableServiceApiClient.DefaultScopes);
-                }
-                channelCredentials = credentials.ToChannelCredentials();
-            }
-            var clientCount = clientCreationSettings?.ClientCount ?? Environment.ProcessorCount;
-
-            var settings = clientCreationSettings?.BigtableServiceApiSettings;
-            var endpoint = clientCreationSettings?.ServiceEndpoint ?? BigtableServiceApiClient.DefaultEndpoint;
-            var clients = new BigtableServiceApiClient[clientCount];
-            var shutdowns = new Func<Task>[clientCount];
-            // Set channel send/recv message size to unlimited.
-            var channelOptions = new[]
-            {
-                new ChannelOption(ChannelOptions.MaxSendMessageLength, -1),
-                new ChannelOption(ChannelOptions.MaxReceiveMessageLength, -1),
-
-                // TODO: Figure out if there's a good way to test this
-                new ChannelOption(ChannelOptions.PrimaryUserAgentString, UserAgent),
-
-                // Use a random arg to prevent sub-channel re-use in gRPC, so each channel uses its own connection.
-                new ChannelOption("sub-channel-separator", Guid.NewGuid().ToString())
-            };
-            // Fill clients[] with BigtableServiceApiClient instances, each with specific channel
-            for (int i = 0; i < clientCount; i++)
-            {
-                var channel = new Channel(endpoint.Host, endpoint.Port, channelCredentials, channelOptions);
-                clients[i] = BigtableServiceApiClient.Create(channel, settings);
-                shutdowns[i] = channel.ShutdownAsync;
-            }
-            Func<Task> shutdown = () => Task.WhenAll(shutdowns.Select(x => x()));
-
-            return new BigtableClientImpl(clients, appProfileId, settings);
+            var client = await BigtableServiceApiClient.CreateAsync(endpoint, settings).ConfigureAwait(false);
+            return new BigtableClientImpl(client, appProfileId, settings);
         }
 
         /// <summary>
         /// Synchronously creates a <see cref="BigtableClient"/>, applying defaults for all unspecified settings,
-        /// and creating a channel connecting to the given endpoint with application default credentials where
+        /// and creating channels connecting to the given endpoint with application default credentials where
         /// necessary.
         /// </summary>
-        /// <param name="clientCreationSettings">Optional. <see cref="ClientCreationSettings"/> specifying how to create
-        /// <see cref="BigtableServiceApiClient"/>s.</param>
+        /// <param name="endpoint"></param>
```

Comment:
Documentation again. (And the method below. I won't add any more comments like this.)